mteb/abstasks/AbsTaskMultilabelClassification.py

```diff
@@ -181,11 +182,13 @@ def _evaluate_subset(
         except ValueError:
             logger.warning("Couldn't subsample, continuing with the entire test set.")
 
-        X_test = model.encode(
-            test_text,
-            model=model,
-            task_name=self.metadata.name,
-            **encode_kwargs,
+        X_test = normalize_embeddings_to_numpy(
+            model.encode(
+                test_text,
+                model=model,
```

Comment:
I accidentally left `model=model` here, and it should be removed.

Reply:
removed